setup.py

```diff
@@ -27,6 +27,7 @@
           "enum"],
```

Comment:
Do we even need this `enum` package, here and in tests?  It doesn't seem to be used.  Would prefer the backported `enum34` version, if we do need.
If it can just be removed, that can be part of this PR, however, if rework is required, then we can leave it as is.